azure/requests.yaml

```diff
@@ -1,4 +1,12 @@
 releases:
+    - name: "> 13.0.0-beta1"
```

Comment:
@giantswarm/team-celestial Is this okay to set the beta version as a threshold?

Reply:
I don't see why not

Reply:
Okay, let's go with this!